addon/templates/components/frost-password.hbs

```diff
@@ -3,6 +3,7 @@
   class='frost-password-input'
   disabled=disabled
   form=form
+  hook=(concat hook '-text')
```

Comment:
Frost-text adds ‘-input’ though to its input field. So internally in password you can get to it with <hook>-text-input

From: Steven Glanzer <notifications@github.com<mailto:notifications@github.com>>
Reply-To: ciena-frost/ember-frost-core <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 28, 2016 at 1:24 PM
To: ciena-frost/ember-frost-core <ember-frost-core@noreply.github.com<mailto:ember-frost-core@noreply.github.com>>
Cc: Panthaky Roxanne <rpanthak@ciena.com<mailto:rpanthak@ciena.com>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
Subject: Re: [ciena-frost/ember-frost-core] Adding ember-hook to core components (#153)


In addon/templates/components/frost-password.hbs<https://github.com/ciena-frost/ember-frost-core/pull/153#discussion_r72664515>:

> @@ -3,6 +3,7 @@
>    class='frost-password-input'
>    disabled=disabled
>    form=form
> +  hook=(concat hook '-text')


Can you make this -input instead?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/ciena-frost/ember-frost-core/pull/153/files/e95e298f3ce82b73563134e3d8a99a83eaf2dc0c#r72664515>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AATH5r-xRKcrrX2PfD9I_9RE_onxb0bkks5qaOXigaJpZM4JWbYH>.